packages/tests/tests/ephemeral.node.spec.ts

```diff
@@ -186,7 +186,16 @@ describe("Waku Message Ephemeral field", function () {
 
     expect(messages?.length).eq(0);
 
-    await tearDownNodes([], [waku1, waku2]);
+    const additionalServiceNodes = await ServiceNodesFleet.createAndRun(
+      this.ctx,
+      0,
+      false,
+      {
+        clusterId: ClusterId,
+        contentTopics: [TestContentTopic, AsymContentTopic, SymContentTopic]
+      }
+    );
+    await teardownNodesWithRedundancy(additionalServiceNodes, [waku1, waku2]);
```

Comment:
it seems wrong: `additionalServiceNodes` is created just to be teared down at the next line

Reply:
fair - f5ac8c1390e01511ad978e14e614e17a73630c2d